git-remote-hg

```diff
@@ -332,7 +332,7 @@ class BranchMap(object):
                 sha1 = store.changeset_ref(head)
                 if not sha1:
                     continue
-                extra = store.read_changeset_data(sha1).get('extra')
+                extra = store.read_changeset_data(sha1)
```

Comment:
Also, if line 332 returns a value, then read_changeset_data should return something. Your error suggests that the cinnabar metadata is inconsistent, and the question is how that happened.